setup.py

```diff
@@ -37,6 +37,7 @@
         'sphinx-rtd-theme>=0.1.9',
         'sphinxcontrib-httpdomain>=1.5.0',
         'sphinxcontrib-openapi>=0.3.0',
+        'sphinxcontrib-redoc>=1.5.1',
```

Comment:
Can you please update to latest `reana-commons`, and move `pytest-reana` dependency to `tests_require`?